Alkiiro/refactor-tractor
# Post

###What's this PR do?
This PR adds a the structure for our post requests in HTML. There are now three more blocks that we will add functionality to post later. 

###Where should the reviewer start?
The reviewer can pull down the repo and make sure that the HTML blocks show up! No functionality works yet.

###How should this be manually tested?
Pull it down and check the tests as well as the html elements.

###Any background context you want to provide?
Post functionality will be split up into Hydration (Nick), Sleep (Foster), and Activity (Alex or combined effort).
